libs/accounts/recovery-phone/src/lib/recovery-phone.manager.ts

```diff
@@ -57,4 +57,19 @@ export class RecoveryPhoneManager {
       throw err;
     }
   }
+
+  async storeUnconfirmed(
+    uid: string,
+    phoneNumber: string,
+    code: string,
+    setup: boolean
+  ) {
+    // TODO: Provide real implementation FXA-10346
+    console.log('TODO: implement storeUnconfirmed', {
```

Comment:
@vbudhram Did this for now. Let me know when FXA-10346 is ready.

Reply:
@dschom Just pushed https://github.com/mozilla/fxa/pull/18012 if you wanted to take a look.